spec/unit/mixin/openssl_helper_spec.rb

```diff
@@ -665,7 +668,7 @@
   end
 
   describe "#renew_x509_crl" do
-    before(:all) do
```

Comment:
`before(:all)` is usually used when the test setup takes a long time or can only be done once. Is this necessary to get the test to pass? Could the line added in this PR be moved to its own `before(:each)` block if that is the problem?